packages/configurationmap/index.ts

```diff
@@ -1 +1,23 @@
 export { default as ConfigurationMap } from "./components/ConfigurationMap";
+export {
+  default as ConfigurationMapSection
```

Comment:
i like skipping the prefix of `ConfigurationMap*` to be able to either 

```typescript
import { Section } from "components/ConfigurationMap"

// OR

import * as CMap from "components/ConfigurationMap"

render(
  <CMap.Section />
)
```

but that's personal preference. no action item here.